lightning/src/offers/invoice.rs

```diff
@@ -1171,7 +1171,7 @@ tlv_stream!(InvoiceTlvStream, InvoiceTlvStreamRef, 160..240, {
 	(174, features: (Bolt12InvoiceFeatures, WithoutLength)),
 	(176, node_id: PublicKey),
 	// Only present in `StaticInvoice`s.
-	(238, message_paths: (Vec<BlindedPath>, WithoutLength)),
+	(238, async_receive_message_paths: (Vec<BlindedPath>, WithoutLength)),
```

Comment:
If the spec uses a different name, I'd use that here instead.

Reply:
The spec uses `invoice_message_paths`. In the past we've tended to choose our own names if we think they're an improvement on the spec, such as `scid_privacy` vs `scid_alias`.

Reply:
Ah, I wasn't aware that was the reasoning behind that choice.